shop_online_api/app/controllers/api/v1/categories_controller.rb

```diff
@@ -0,0 +1,34 @@
+class Api::V1::CategoriesController < ApplicationController
+  def index
+    if params[:check] == 'sub'
+      @cate_id = Category.where(parent_id: 0).map(&:id)
```

Comment:
1. comment to clarify why you search for parent_id = 0
2. Local variable or instance variable should be used?